build/edge/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM golang:1.12.1-alpine3.9 AS builder
+
+COPY . /go/src/github.com/kubeedge/kubeedge
+
+RUN apk --no-cache update && \
+apk --no-cache upgrade && \
+apk --no-cache add build-base linux-headers sqlite-dev && \
+CGO_ENABLED=1 go build -v -o /usr/local/bin/edge_core -ldflags="-w -s -extldflags -static" \
```

Comment:
there is an open PR to compile a miniature edge_core binary using upx. Just for your reference : https://github.com/kubeedge/kubeedge/pull/262

Reply:
Yes, we'v shared the idea at #208 .

Reply:
upx is just an option, it's not default. I don't like upx very much as well.